.github/workflows/release.yaml

```diff
@@ -86,7 +86,7 @@ jobs:
           path: dist
       - name: "Archive binary"
         run: |
-          ARCHIVE_FILE=ruff-x86_64-apple-darwin.tar.gz
+          ARCHIVE_FILE=ruff-${{ inputs.tag }}-x86_64-apple-darwin.tar.gz
```

Comment:
Above, tag says `"The version to tag, without the leading 'v'. If omitted, will initiate a dry run (no uploads)."` Do you know if that's still true? Can it be omitted? And, if it is omitted, what happens here?

Reply:
It can still be omitted, in which case `${{ inputs.tag }}` expands to an empty string. That specific example would result in `ruff--x86_64-apple-darwin.tar.gz` in dry-run mode. You can see an example in [this workflow run](https://github.com/tobbez/ruff/actions/runs/7093701040).

Reply:
It'd be nice to omit the second dash, but that's just a debugging workflow so not critical.